Bottom: 9d6c66cb15c181e3f632faf74d76debb5f07eb7c
Top:    0ec304f8f75da916e81c6c67c1e650e167aa023e
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-09-04 16:51:28 +0800

Refresh of vhost_net-basic-batch

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index ac0a914..924b2b5 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -534,7 +534,7 @@ static void handle_tx(struct vhost_net *net)
 
 			zcopy_used = zcopy && len >= VHOST_GOODCOPY_LEN
 				     && (nvq->upend_idx + 1) % UIO_MAXIOV !=
-				         nvq->done_idx
+					nvq->done_idx
 				     && vhost_net_tx_select_zcopy(net);
 
 			/* use msg_control to pass vhost zerocopy ubuf info to skb */
